cosmos/scripts/custom-script.sh

```diff
@@ -27,10 +53,10 @@ insertstart=$((YCSB_RECORD_COUNT * (MACHINE_INDEX - 1)))
 recordcount=$((YCSB_RECORD_COUNT * MACHINE_INDEX))
 # Record count for Run. Since we run read workload after load this is the total number of records loaded by all VMs/clients during load.
 totalrecordcount=$((YCSB_RECORD_COUNT * VM_COUNT))
-benchmarkname=ycsbbenchmarking
+benchmarkname=$PROJECT_NAME
 if [ $WAIT_FOR_FAULT_TO_START_IN_SEC -gt 0 ] && [ $DURATION_OF_FAULT_IN_SEC -gt 0 ]; then
   fault=true
-  benchmarkname=ycsbwithfault
+  benchmarkname="{$PROJECT_NAME}withfault"
```

Comment:
For bash variable interpolation to work it should be ${PROJECT_NAME}withfault

Reply:
Also, since we are using the project name in the container name. Need to make sure that it complies with the following:
https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names 

Let's discuss how to handle this.

Reply:
Discussed in the call. Updated the code to include a note for user to not use any hyphens in the project name.